src/protocols/vnc/vnc.c

```diff
@@ -133,6 +141,27 @@ rfbClient* guac_vnc_get_client(guac_client* client) {
     /* TLS Locking and Unlocking */
     rfb_client->LockWriteToTLS = guac_vnc_lock_write_to_tls;
     rfb_client->UnlockWriteToTLS = guac_vnc_unlock_write_to_tls;
+#endif
+    
+#ifdef LIBVNCSERVER_WITH_CLIENT_GCRYPT
```

Comment:
Where is this (conditionally) defined?

Reply:
rfbconfig.h:
```
/* Enable support for libgcrypt in libvncclient */
#ifndef LIBVNCSERVER_WITH_CLIENT_GCRYPT 
#define LIBVNCSERVER_WITH_CLIENT_GCRYPT  1 
#endif
```

which is then included in rfbproto.h.

Reply:
Sure - that's a great idea.

I was initially just looking for where the macro was coming from, as there were no changes to `configure.ac` and my quick grep of libvncclient code didn't reveal anything, but this sounds excellent. It avoids any potential for what we saw with the accidental omission of our own `config.h` in handling of FreeRDP behavior.

Reply:
Should be so, now.